pkg/cluster/logs/logs.go

```diff
@@ -105,6 +111,10 @@ func Collect(nodes []nodes.Node, dir string) error {
 					node.Command("journalctl", "--no-pager", "-u", "containerd.service"),
 					filepath.Join(name, "containerd.log"),
 				),
+				execUntarToPathFn(
+					node.Command("tar", "--hard-dereference", "-C", "/var/log", "-chf", "-", "."),
```

Comment:
is it possible to see output of which file is being tarred? i saw some examples online but it didn't seem a matter of adding a tar flag only.

Reply:
adding a -v to the tar command should output that info, but IIRC all the output is redirected to the reader so it can untar it, also I think that there were problems if we allocate a tty to run the command, 
```
DEBU[15:02:00] Running: /usr/bin/docker [docker exec --privileged kind-control-plane tar --hard-dereference -C /var/log -zchf - .]
```
Ben has a much better understanding, it would be great to know his opinion

Reply:
ok, just wanted to point out that there are way to many reports on the web of the tar command hanging for a variety of reasons e.g. cannot read file.

Reply:
that´s why I moved it to their own thread, ... should we add a timeout to the export logs and close if exceeded?

Reply:
```
     --HANG
           hang for SECS seconds (default 3600)
```
not sure if it the actual timeout flag or something else.
http://linuxcommand.org/lc3_man_pages/tar1.html

also not sure if all versions have it.

Reply:
from the stack trace that I´ve uploaded in another comment and looking at one hanging process the tar command seems to exit, can it be possible that is not closing the pipe?